luigi/scheduler.py

```diff
@@ -932,6 +936,31 @@ def disable_worker(self, worker):
     def set_worker_processes(self, worker, n):
         self._state.get_worker(worker).add_rpc_message('set_worker_processes', n=n)
 
+    @rpc_method()
+    def send_scheduler_message(self, worker, task, message):
+        if not self._config.send_messages:
+            return {"messageId": None}
+
+        message_id = str(uuid.uuid4())
+        self._state.get_worker(worker).add_rpc_message('dispatch_scheduler_message', task_id=task,
+                                                       message_id=message_id, message=message)
+
+        return {"messageId": message_id}
```

Comment:
yuck, I just realize that we mix camelCase and snake_case in the key-names. But it seems snake_case is more used. Maybe use that?

Reply:
✔︎